unix/w0vncserver/portals/RemoteDesktop.cxx

```diff
@@ -51,6 +57,12 @@ core::BoolParameter localCursor("experimentalPortalLocalCursor",
                                 "[EXPERIMENTAL] Render cursor locally",
                                 false);
 
+core::EnumParameter askDisplayChoice("portalAskDisplayChoice",
```

Comment:
Why the "portal" prefix? Seems like technical detail that end users shouldn't need to know.